INFO - 2021-08-31 14:02:15,379 - new_apply_order.py[line:518]: 开始第 borrowme【1】次进件……
INFO - 2021-08-31 14:02:15,379 - new_apply_order.py[line:79]: ======【nga测试环境】生成手机号：01630389735 ======
INFO - 2021-08-31 14:02:15,746 - new_apply_order.py[line:86]: 01630389735 检测用户存在成功！
INFO - 2021-08-31 14:02:15,800 - new_apply_order.py[line:93]: 01630389735 短信发送成功！
INFO - 2021-08-31 14:02:15,841 - new_apply_order.py[line:103]: 01630389735 注册成功！
INFO - 2021-08-31 14:02:15,889 - new_apply_order.py[line:113]: 01630389735 登录短信发送成功！
ERROR - 2021-08-31 14:02:15,936 - new_apply_order.py[line:564]: module 'common.params_init' has no attribute 'data_login'
ERROR - 2021-08-31 14:02:15,938 - new_apply_order.py[line:565]: None
